src/hl7v2/config.clj

```diff
@@ -0,0 +1,5 @@
+(ns hl7v2.config)
+
+(def config
+  (delay
+    {:standard-dir "tmp"}))
```

Comment:
Este delay como tal qué hace, porque lo he visto pero en otro contexto, por ejemplo `(def my-delay (delay (println "Procesando...") 100))`. Aquí es como si invocase el cuerpo de la la función una vez y lo que retorna lo guarda como si fuera caché. Pero aquí el body es ese map, así que no comprendo qué sucede.

Reply:
In this case, we can remove the `(delay`, because we are not processing or computing anything. The `(delay` is useful when we need to do some processing to delay the execution of the computation until the first usage of the var. It is not a good idea to do anything expensive as a top-level form in the namespace, such as accessing files, initializing database connections, requesting, parsing, or similar tasks, which are wrapped with `(delay` so the namespace can be loaded efficiently. I plan to load config from a file and/or env vars, so I wrapped the map in the delay to have the client code dereferencing the config.